### PR TITLE
Improve Copilot + local AI setup instructions

### PR DESCRIPTION
**Pull request by azigler**
I noticed [this comment from a user](https://github.com/mattermost/mattermost-plugin-ai/issues/143#issuecomment-2358807158) that had trouble configuring Copilot with Ollama and realized that the documentation instructions do not include the specific URL instructions to access the OpenAI-compatible API. This PR clarifies that instruction, in line with what's taught in [the Academy course](https://academy.mattermost.com/p/new-mattermost-copilot-multi-llm-setup-usage).

### CODE REVIEW

File: source/configure/enable-copilot.rst [hunk 1]
```diff
@@ -82,7 +82,7 @@ Configure a large language model (LLM) for your Copilot integration by going to
 
   1. Deploy your model, for example, on `Ollama <https://ollama.com/>`_.
   2. Select **OpenAI Compatible** in the **AI Service** dropdown.
-  3. Enter the URL to your AI service from your Mattermost deployment in the **API URL** field.
+  3. Enter the URL to your AI service from your Mattermost deployment in the **API URL** field. Be sure to include the port, and append `/v1` to the end of the URL. (e.g., `http://localhost:11434/v1` for Ollama)
```
Comment (cwarnermm) [inline source/configure/enable-copilot.rst:85]:
```suggestion
  3. Enter the URL to your AI service from your Mattermost deployment in the **API URL** field. Be sure to include the port, and append ``/v1`` to the end of the URL. (e.g., ``http://localhost:11434/v1`` for Ollama)
```

**Review by cwarnermm — APPROVED** [review]
Thank you, @azigler!

**Comment by azigler** [comment]
@crspeller Should we include a recommendation for disabling tools? Is that a broad recommendation? If so, should we revisit the option and it's default setting? As a minor note, it's a bit confusing that the field is "disable" and has a true/false (which means the radio toggle is semantically opposite from the end result, which might confuse users). What about "Enable Tools" and it has a default setting to whatever you think is most appropriate?

**Comment by cwarnermm** [comment]
Friendly reminder that this docs PR will benefit from your review, @crspeller.

File: source/configure/enable-copilot.rst [hunk 2]
```diff
@@ -82,7 +82,7 @@ Configure a large language model (LLM) for your Copilot integration by going to
 
   1. Deploy your model, for example, on `Ollama <https://ollama.com/>`_.
   2. Select **OpenAI Compatible** in the **AI Service** dropdown.
-  3. Enter the URL to your AI service from your Mattermost deployment in the **API URL** field.
+  3. Enter the URL to your AI service from your Mattermost deployment in the **API URL** field. Be sure to include the port, and append ``/v1`` to the end of the URL. (e.g., ``http://localhost:11434/v1`` for Ollama)
```
Comment (crspeller) [inline source/configure/enable-copilot.rst:85]: The instruction to add `/v1` is Ollama specific not a general instruction for all OpenAI compatable APIs. I don't think that is clear here.

**Review by crspeller — CHANGES_REQUESTED** [review]

**Comment by cwarnermm** [comment]
@azigler - Are you open to adding a direct link from our LLM admin docs to LLM setup training via Academy? You can use https://github.com/mattermost/docs/pull/7465 as a reference to do so.

**Comment by azigler** [comment]
@crspeller Clarified that.

@cwarnermm Added that!

Good to go! 👍

**Review by crspeller — APPROVED** [review]

**Pull request merged**